profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.admin.inc

```diff
@@ -0,0 +1,597 @@
+<?php
+/**
+ * @file
+ * Nexteuropa Newsroom administration page.
+ */
+
+/**
+ * Nexteuropa newsroom admin form.
+ */
+function nexteuropa_newsroom_admin_settings($form, &$form_state) {
+  // Be aware about the selected content types in the fields fieldset.
+  $sel_ct = NewsroomHelper::getFieldInstances('select');
```

Comment:
Store the instances in database instead of always asking them via very heavy operands - this helper function is quite useless and makes it slower.